src/services/localizations/index.ts

```diff
@@ -0,0 +1,29 @@
+import { OAuth1Client } from '@extrahorizon/javascript-sdk';
+import { yellow } from 'chalk';
+import { PartialLocalization } from '../../repositories/localizations';
+import { readFiles } from './readFiles';
+import { syncLocalizations } from './syncLocalizations';
+
+export async function sync(sdk: OAuth1Client, path: string) {
+  console.log(yellow(`Synchronizing localizations from ${path}`));
+  const localizations = readFiles(path);
+
+  if (localizations.length < 1) {
+    console.log(yellow('No localizations found'));
+    return;
+  }
+
+  assertDefaultLanguageSetForAll(localizations);
+
+  await syncLocalizations(sdk, localizations);
+}
+
+function assertDefaultLanguageSetForAll(localizations: PartialLocalization[]) {
+  const defaultLanguage = 'EN';
```

Comment:
(Question) Is this how the service works? 🤔 It requires a default English key before you can add multiple languages?